osd_whats_new/osd-whats-new.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="osd-whats-new"]
+= What's new with {product-title}
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: osd-whats-new
+
+toc::[]
+
+With its foundation in Kubernetes, {product-title} is a complete {OCP} cluster provided as a cloud service, configured for high availability, and dedicated to a single customer.
+
+{product-title} is professionally managed by Red Hat and hosted on {GCP} or {AWS}. Each {product-title} cluster includes a fully managed link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane[control plane] (Control and Infrastructure nodes), application nodes, installation and management by Red Hat Site Reliability Engineers (SRE), premium Red Hat Support, and cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
+
+{product-title} clusters are available on the link:https://console.redhat.com/openshift[Hybrid Cloud Console]. With the Red Hat {cluster-manager} application, you can deploy {product-title} clusters to either on-premises or cloud environments.
+
+[id="osd-new-changes-and-updates_{context}"]
+== New changes and updates
+
+[id="osd-q2-2024_{context}"]
+=== Q2 2024
+
+* **CSI Operator update.** {product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Compute Platform (GCP) Filestore Storage. For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#persistent-storage-csi-google-cloud-file-overview[Google Compute Platform Filestore CSI Driver Operator].
+
+* **Support for new GCP instances.** {product-title} now supports more worker node types and sizes on Google Cloud. For more information, see xref:../osd_architecture/osd_policy/osd-service-definition.adoc#gcp-compute-types_osd-service-definition[Google Cloud compute types].
+
+[id="osd-q1-2024_{context}"]
+=== Q1 2024
+
+* **{product-title} regional availability update.** {product-title} is now available in the following region:
```

Comment:
I wonder if we need to word this to make it clearer that this newly available vs seeming like the only avail region?